code/script.js

```diff
@@ -1,47 +1,235 @@
+ const greeting = () => {
+   showMessage(`Hello there, What's your name?`, 'bot')
+ }
+  
+// Just to check it out, change 'bot' to 'user' here 👆
+
+
+ const handleInput = (event) => {
+   currentQuestion++
+   event.preventDefault()
+
+   console.log('our currentQuestion variable is:', currentQuestion)
+   if (currentQuestion === 1) {
+    handleNameQuestion()}
+    else if (currentQuestion === 2) {
+      languageQuestion()}
+   }
+
+  // Questions asked 
+ 
+  const languageQuestion = (name) => {
+    showMessage(`Hi ${name}, Welcome by learning Swedish which subject do you want to learn?`, 'bot')
+
+    inputWrapper.innerHTML = `
+      <button id="food">Food🥗</button>
+      <button id="greetings">Greetings👋</button>
+      <button id="jobs">Jobs👨‍⚖️</button>
+    `
+    document 
+      .getElementById('food')
+      .addEventListener('click', () => {askForType('food')})
+    document
+      .getElementById('greetings')
+      .addEventListener('click', () => {askForType('greetings')})
+    document
+      .getElementById('jobs')
+      .addEventListener('click', () => {askForType('jobs')})
+
+      
+
+      currentQuestion++
+      console.log('Current question set to', currentQuestion)
+   }
+
+   
+
+  //Question Functions
+
+    const handleNameQuestion = () => {
+      const name = inputValue.value
+
+      showMessage(`My name is ${name}`, 'user')
+      inputValue.value = ''   
+
+      setTimeout(() => languageQuestion(name), 1500)
+    }
+
+   
+    const askForType = (subject) => {
+      if (subject === 'food') {
+        showMessage(`I choose ${subject}`, 'user')
+        showMessage(`Ok food it is! Translate: I like strawberries in Swedish.`, 'bot')
+        // form.innerHTML = ''
+        inputWrapper.innerHTML = `          
+          <button id="good">Jag tycker om jordgubber</button>
+          <button id="false">Ik hou van aardbeien</button>
+          <button id="false0">Jag älskar jordgubber</button>
+       `
+
+        document 
+        .getElementById('good')
+        .addEventListener('click', () => {showMessage('That was the right answer', 'bot')
+
+        {inputWrapper.innerHTML = `
+        <button id="retry">Try again...</button>
+        <button id="end">End Programming</button>
+            `
+            
+
+            document
+            .getElementById('retry')
+            .addEventListener('click', () => {location.reload()})
+            document
+            .getElementById('end')
+            .addEventListener('click', () => {showMessage('Congratiolations on completing the program!' , 'bot')})
+       }
+      })
+
+
+
+
+        document
+        .getElementById('false')
+        .addEventListener('click', () => {showMessage('Sorry try again' , 'bot')})
+        document
+        .getElementById('false0')
+        .addEventListener('click', () => {showMessage('Sorry try again' , 'bot')})
+
+
+
+        currentQuestion++
+        console.log('Current question is now', currentQuestion)
```

Comment:
Really good use of `console.log` to keep track of where you are during development. Perhaps we can comment these out for the final product.